hangman/lib/hangman/application.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Hangman.Application do
+  use Application
+
+  def start(_type, _args) do
+    import Supervisor.Spec, warn: false
+
+    children = [
```

Comment:
# +5

You're the first person who implemented a separate server to hold the game state. Well done.

However, I didn't give you the full 10 extra because you could have supervised it differently. 

The idea of keeping separate state is that, if the code crashes, the state lives on. If the state crashes, then everything should die. So really you should supervise this using `:rest_for_one` and make the store the first child.